### ISSUE TITLE
User can switch between two configurations

### ISSUE THREAD

**Issue by sveneh** [cubedro/eth-netstats#6]
config / monitored nodes are stored in config/nodes.js

how to switch between develop nodes and live nodes?

Ideally I can start with cmd line pointing to a different config directory. This option should be exposed also to the docker image, so I can do something like this

```
docker run eth-netstat --config config-live 
docker run eth-netstat --config config-devel 
```

**Comment by cubedro** [comment]
I've added "config/nodes.default.js" with local node and instructions to add nodes. In order to have a local config file ignored by git users can duplicate the file and rename it to "config/nodes.js" and it will be loaded as the main config file. Fixed in commit 3c2716653b8045f959f491b8c3dc16e82258c6bd

**Comment by cubedro** [comment]
Not needed anymore since the RPC services connect directly to the server.
